examples/ci_test.rs

```diff
@@ -139,14 +137,14 @@ fn simulate_churn(mut nodes: Vec<NodeProcess>,
 
         loop {
             {
-                let &(ref lock, ref cvar) = &*stop_flg;
+                let &(ref lock, ref condition_variable) = &*stop_flag;
```

Comment:
Even the standard library calls it `Condvar` - shall we go with `condvar` as a compromise?